extensions/yaml-configuration/runtime/src/main/java/io/quarkus/yaml/configuration/runtime/JarApplicationYamlConfigSource.java

```diff
@@ -0,0 +1,30 @@
+package io.quarkus.yaml.configuration.runtime;
+
+import java.io.InputStream;
+
+public class JarApplicationYamlConfigSource extends YamlConfigSource {
+
+    private static final long serialVersionUID = 2819560924855482475L;
+
+    private static final String YAML_FILE = "application.yml";
+    private static final int PRIORITY = 270;
```

Comment:
The properties-based version of this config source priority is `250`. See `ApplicationPropertiesConfigSource$InJar`.